typings/InAppBillingHelper.android.d.ts

```diff
@@ -6,7 +6,7 @@ declare namespace com {
             export class InAppBillingHelper {
                 public static BUY_INTENT_REQUEST_CODE: number;
                 
-                constructor(context: android.content.Context, productIdentifiers: Array<string>);
+                constructor(context: android.content.Context, productIdentifiers: Array<string>, subs: boolean);
```

Comment:
I don't like having ability to filter either. Since on iOS there is no way to do this and it always returns both in-app purchases and subs, this will be an inconsistency. Also having git this way there is no way for the user to show both subs and in-app purchases. I think it is better to just return both and eliminate the `subs` parameter.